_posts/2022-12-01-trino-summit-2022-sk-telecom-recap.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: "Journey to Iceberg with Trino"
+author: "JaeChang Song, Jennifer Oh, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/apple.jpg
+---
+
+This post comes from [the later half of Trino Summit 2022 session]({% post_url 
+2022-11-21-trino-summit-2022-recap %}). Our friends JaeChang and Jennifer from
+SK Telecom, traveled across the globe from South Korea to join us in person! SK
+Telecom recently had some issues scaling Trino on the Hive model, among other
+issues that come with Hive. While some initial tweaking helped speed things up,
+it ultimately never solved the problem. SK Telecom initially tested performance
+after moving on Iceberg with some very impressive results. They talk about their
+journey to Iceberg via Trino.
+
+<!--more-->
+
+{% youtube V9_aPLXATh8 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@SK-Telecom.pdf">
+  Check out the slides here!
+</a>
+
+## Recap
+
+SK Telecom is a South Korean Telecom company that has built and operated
+open source based on-premise data platform for manufacturing data to determine
+manufacturing yield since 2015. SK Telecom's goal has always been to build an
+observable federated data platform on open source software at scale. 
+
+SK Telecom manages on-premise Hadoop clusters to store their data. SK Telecom
+started using Presto in 2016, and shifted to Trino in 2021. To run batch queries
+on their warehouse, Trino workers are deployed on HDFS data nodes. There is also
+an adhoc Trino cluster deployed to manage federated queries over multiple data
+silos from an array of disparate data sources. Previously, they used tools like 
+[distcp](https://hadoop.apache.org/docs/stable/hadoop-distcp/DistCp.html) to
+make data available in one center. This was one of the slow and brittle process
+that Trino replaced. They chose Trino because it simplified querying novel big
+data systems and combining that data more commonplace systems for their users.
+
+As Trino adoption grew within the company to the tune of 300 requests per
+minute, they eventually faced challenges scaling. Not only were the number of
+requests growing, but users increased the range of data they wanted to query.
```

Comment:
"users increased the range of data" makes it sounds like individual users chose to query more data at once, which doesn't seem like what you mean to convey.

Suggest "the range of data being queried grew as well" or similar